src/annotator/plugin/toolbar.coffee

```diff
@@ -39,6 +49,7 @@ module.exports = class Toolbar extends Plugin
           @toolbar.find('[name=sidebar-close]').hide();
     ,
       "title": "Toggle or Resize Sidebar"
+      "ariaPressed": !!window.hypothesisConfig().openSidebar
```

Comment:
This has the same issue as the code above in that it could potentially get out of sync with the actual sidebar open/closed state if:

- The `openSidebar` configuration handling changes elsewhere
- Some additional situation is introduced in which the `openSidebar` configuration is overruled or can get changed during startup after this code runs